$NetBSD: patch-configure,v 1.1 2014/12/30 15:39:51 wiz Exp $
pkgsrc doesn't use lib64 or targetdir so don't set it.
fixes build on linux.
http://sf.net/p/qore/code/6817/
--- configure.orig	2014-10-31 18:50:53.724785164 +0000
+++ configure
@@ -17467,14 +17467,6 @@ if test "$enable_64bit" = "yes"; then
       *linux*)	if test "$GXX" = "yes"; then
 		   CXXFLAGS="$CXXFLAGS -m64"
 		fi
-		# only set LIBSUFFIX=64 if /usr/lib64 exists
-		if test -d /usr/lib64; then
-		   LIBSUFFIX=64
-                fi
-		target="${host_cpu}-${host_os}"
-		if test -d /usr/lib/$target; then
-		   LIBSUFFIX=/$target
-                fi
 
 		;;
       *hpux*)	if test "$host_cpu" = "ia64"; then
